packages/components/src/DateTimePickers/InputTimePicker/InputTimePicker.component.js

```diff
@@ -21,14 +22,19 @@ const PROPS_TO_OMIT_FOR_INPUT = [
 	'onChange',
 ];
 
-export default function InputTimePicker(props) {
+function InputTimePicker(props) {
```

Comment:
This is for later compose InputDateTimePicker, so it can use context provided by InputDateTimePicker